### PR TITLE
Fix project list

### PR DESCRIPTION
**Pull request by agaspardcilia**
Will now only display owned repositories for user's company. This will prevent the user from having every public repository available in the frontend project select.

### CODE REVIEW

File: src/main/java/io/github/jhipster/online/service/GithubService.java [hunk 1]
```diff
@@ -145,23 +158,14 @@ public User getSyncedUserFromGitProvider(User user) throws Exception {
                 gitCompanyRepository.save(organization);
                 organizations.add(organization);
             }
-            try {
-                syncCompanyGitProjects(gitHub, organization);
-            } catch (IOException e) {
-                log.error("Could not sync GitHub repositories for user `{}`: {}", user.getLogin(), e.getMessage());
-            }
+            organization.setGitProjects(new ArrayList<>(myOrganizations.get(organizationName).getRepositories().keySet()));
+            System.out.println(organization.getGitProjects());
```
Comment (jdubois) [inline src/main/java/io/github/jhipster/online/service/GithubService.java]: please remove sysout

File: src/main/docker/Dockerfile [hunk 2]
```diff
@@ -4,6 +4,10 @@ ENV SPRING_OUTPUT_ANSI_ENABLED=ALWAYS \
     JHIPSTER_SLEEP=0 \
     JAVA_OPTS=""
 
+RUN apk update && \
```
Comment (jdubois) [inline src/main/docker/Dockerfile]: Next time please do a separate PR for this, because this is a different issue

Comment (agaspardcilia) [reply src/main/docker/Dockerfile]: You're right, I can move this to another PR if it makes things easier for you.
